lib/pengine/unpack.c

```diff
@@ -2687,15 +2690,15 @@ unpack_migrate_to_success(pe_resource_t *rsc, pe_node_t *node, xmlNode *xml_op,
         if (target_node && target_node->details->online) {
             pe_rsc_trace(rsc, "Marking active on %s %p %d", target, target_node,
                          target_node->details->online);
-            native_add_running(rsc, target_node, data_set);
+            native_add_running(rsc, target_node, data_set, action_fail_ignore);
```

Comment:
Here, the migration failed, but we consider the resource active on the migration target, because it might be. However since it's failed, I don't think we want to count it to the node priority. Any other action_fail_ should be sufficient for that purpose. If you don't like using an arbitrary action_fail_ value, we could just use a bool for the function argument where true is failed.

Reply:
Okay!

Set failures other than action_fail_ignore.